checker/build.xml

```diff
@@ -510,6 +520,15 @@
     <!-- Used to guarantee that targets executed from other ant scripts do not cause unnecessary rebuilds -->
     <target name="all-tests-nojar" depends="build-tests"
             description="Run tests for all checkers, WITHOUT building anything">
+
```

Comment:
Can we reduce the duplication, which existed before your changes, and somehow merge the two "junit" tasks?